source/contracts/reporting/Universe.sol

```diff
@@ -125,6 +127,7 @@ contract Universe is DelegationTarget, ITyped, Initializable, IUniverse {
     function getOrCreateChildUniverse(bytes32 _parentPayoutDistributionHash) public returns (IUniverse) {
         if (childUniverses[_parentPayoutDistributionHash] == address(0)) {
             childUniverses[_parentPayoutDistributionHash] = UniverseFactory(controller.lookup("UniverseFactory")).createUniverse(controller, this, _parentPayoutDistributionHash);
+            Augur(controller.lookup("Augur")).logUniverseCreated(childUniverses[_parentPayoutDistributionHash]);
```

Comment:
This feels like it should go in `universe.initialize`.

Reply:
For these cases we can't put these in the initialize because it hasn't been added into the universe tree yet (Both for this and the market creation log)